template/package.json.js

```diff
@@ -0,0 +1,50 @@
+
+import { kebabCase } from '../utils/index';
+import { File } from '@asyncapi/generator-react-sdk';
+
+export default function packageFile({ asyncapi }) {
+  let name = 'NatsTSclient';
+  let version = '1.0.0';
+  let description = '';
+  if (asyncapi.info().title()) {
+    name = kebabCase(asyncapi.info().title());
+  }
+  if (asyncapi.info().version()) {
+    version = asyncapi.info().version();
+  }
+  if (asyncapi.info().description()) {
+    description = asyncapi.info().description();
+  }
+  return <File name={'package.json'}>
+    {
+      `
```

Comment:
Creating parsed json in js file you first create normal js object `{ name: name, description: description... }` etc, and then use `JSON.parse(obj, null, 2)` :)

Reply:
Good idea :)